cmake/Config.cmake.in

```diff
@@ -0,0 +1,26 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
+check_required_components("@PROJECT_NAME@")
+
+if(@OpenMP_CXX_FOUND@ OR @OPENMP_FOUND@)
+  find_package(OpenMP)
```

Comment:
`REQUIRED` missing? If it's pure compiler feature then it's okay. If it depends on libraries installed on system then it should be controlled with option `XGBOOST_USE_OPENMP`.

Reply:
> REQUIRED missing? If it's pure compiler feature then it's okay.

I believe it is always a compiler feature.  The upstream CMakeLists.txt runs `find_package(OpenMP)` and uses that as a test for availability.